samples/charts/tree-map/overview/App.razor

```diff
@@ -24,8 +10,8 @@
                      IdMemberPath="Name"
                      LabelMemberPath="Name"
                      ValueMemberPath="Pop"
-                     LayoutType="@TreeLayout"
-                     LayoutOrientation="@TreeOrientation"
+                     LayoutType="TreemapLayoutType.Squarified"
+                     LayoutOrientation="TreemapOrientation.Vertical"
```

Comment:
try removing layout props because those should have the same values by default

Reply:
pending